src/App.tsx

```diff
@@ -1,24 +1,231 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  postTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/Filter';
+import { ErrorType } from './types/Error';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { ToDoList } from './components/ToDoList';
 
-const USER_ID = 0;
+const USER_ID = 6701;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [statusFilter, setStatusFilter] = useState<FilterType>(FilterType.All);
+  const [error, setError] = useState<ErrorType>(ErrorType.None);
+  const [input, setInput] = useState('');
+  const [updating, setUpdeting] = useState(0);
```

Comment:
there could be a user with id = 0, use null instead
```suggestion
  const [updatingId, setUpdatingId] = useState<number | null>(null);
```

Reply:
I think Volodymyr means `todo` not user. But really any entity could has id=0. So it would be better if you handle this case.